src/libcollections/linked_list.rs

```diff
@@ -1376,7 +1376,7 @@ mod tests {
         thread::spawn(move || {
                 check_links(&n);
                 let a: &[_] = &[&1, &2, &3];
-                assert_eq!(a, &n.iter().collect::<Vec<_>>()[..]);
+                assert_eq!(a, &n.iter().collect::<Vec<_>>());
```

Comment:
This is causing Travis to fail -- the second argument isn't getting autoderefed.